docs/installation.md

```diff
@@ -1,20 +1,93 @@
 Installation
 ============
 
-```eval_rst
-.. note:: This section describes how to set up a fully functioning
-          instance of Treeherder. If you only want to hack on the UI,
-          you can just setup a standalone webserver which accesses
-          the server backend using node.js, which is much simpler.
-          See the :doc:`UI development section <ui/installation>`.
-```
-
 Prerequisites
 -------------
 
 * If you are new to Mozilla or the A-Team, read the [A-Team Bootcamp].
-* Install [Git], [Virtualbox] and [Vagrant] (latest versions recommended).
+* Install [Git]
 * Clone the [treeherder repo] from GitHub.
+
+If you only want to hack on the frontend, see the UI Development section below. If you want to hack on the backend or work full-stack, see the [Server and Full-stack Development](#server-and-full-stack-development) section.
+
+UI Development
+==============
+
+To get started:
+
+* Install [Node.js] and [Yarn] (see [package.json] for known compatible versions, listed under `engines`).
+* Run ``yarn install`` to install all dependencies.
+
+Running the standalone development server
+-----------------------------------------
+
+The default development server runs the unminified UI and fetches data from the
+production site. You do not need to set up the Vagrant VM unless making backend changes.
+
+* Start the development server by running:
+
+  ```bash
+  $ yarn start
+  ```
+
```

Comment:
It might be worth adding a ``Note:`` section of some sort that warns the user that when they're running ``yarn start`` that any actions they take (like classifying a job) will affect the real front-end of Treeherder.  So we recommend developing against ``stage`` unless there's something data-specific that must be addressed on ``production``.